tests/src/traffic.rs

```diff
@@ -138,6 +139,7 @@ mod traffic {
     #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
     #[timeout(Duration::from_secs(240))]
     pub async fn outgoing_traffic_udp_with_connect(
```

Comment:
I'm not sure about taking over this test. Is there a good reason not to have a new separate test for the host filter?

Reply:
Nope, it was just convenient. Put the old test back, now we have both.